components/alert/alert.hbs

```diff
@@ -22,7 +22,7 @@
 							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas varius tortor nibh, sit amet tempor nibh finibus et. Aenean eu enim justo. Vestibulum aliquam hendrerit molestie. Mauris malesuada nisi sit amet augue accumsan tincidunt. Maecenas tincidunt, velit ac porttitor pulvinar.
 						</p>
 						<div class="msds-alert__action-container">
-							 <button type="button" class="msds-btn msds-btn--secondary msds-btn--sm">ok, i get it</button>
+							 <button type="button" class="msds-btn msds-btn--secondary msds-btn--sm js-msds-alert-close-click-handler">ok, i get it</button>
```

Comment:
why create a seperate class for an eventhandler?

Reply:
This was implemented by Martin way back when, I am utilizing what already exists